selectrum.el

```diff
@@ -695,6 +740,31 @@ PRED defaults to `minibuffer-completion-predicate'."
       ('current/matches (format "%-6s " (format "%d/%d" current total)))
       (_                ""))))
 
+(defvar display-line-numbers)
```

Comment:
Is this needed? I thought that, being defined in C code, the variable would always be available.

Reply:
Good catch, I have put it there because the test suite also runs for Emacs 25 and the variable was introduced in Emacs 26.